src/test/mir-opt/matches_reduce_branches.rs

```diff
@@ -0,0 +1,13 @@
+// compile-flags: --emit mir
```

Comment:
this line should not be needed, the comment below should "just work" when running the test with `--bless`